wear/src/main/java/io/homeassistant/companion/android/util/ToggleButtonControls.kt

```diff
@@ -0,0 +1,28 @@
+package io.homeassistant.companion.android.util
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.semantics.contentDescription
+import androidx.compose.ui.semantics.semantics
+import androidx.wear.compose.material3.Checkbox
+import androidx.wear.compose.material3.Switch
+import io.homeassistant.companion.android.theme.getCheckboxColors
+import io.homeassistant.companion.android.theme.getSwitchColors
+
+@Composable
+fun GetToggleSwitch(isChecked: Boolean, description: String) = Switch(

```

Comment:
The `description` is always based on `isChecked`, so you could remove it from the constructor + put the if/else directly in here as it isn't used elsewhere.

Reply:
thanks this is now done